conf/gobblin-env.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Set Gobblin specific environment variables here.
+
+# Gobblin work dir
+export GOBBLIN_WORK_DIR=hdfs://localhost:8020/gobblin
+
+# Gobblin job configuration files
+export GOBBLIN_JOB_CONFIG_DIR=$GOBBLIN_WORK_DIR/jobs
```

Comment:
Couple Questions:

1: Since this script is also called from gobblin-standalone.sh one question we should ask, is if the default values for stand-alone mode should write to the local filesystem or to "hdfs://localhost:8020". If we agree that writing to "hdfs://localhost:8020" should be the default behavior, then we need to change the fs.uri parameter in conf/gobblin-standalone.properties

2: Does running this in stand-alone mode work? I think specifying the URI in GOBBLIN_WORK_DIR will cause Gobblin to either throw an error, or do something weird. The writer (see AvroHdfsDataWriter), connects to the filesystem specified by fs.uri, and then writes to the path specified by writer.output.dir. With this change the fs.uri=file:///, while writer.output.dir=hdfs://localhost:8020/gobblin/task-output